### ISSUE TITLE
First time deployment using 'dotnet lambda deploy-function' and passing --function-role always prompts for input

### ISSUE THREAD

**Issue by KeirSweeney** [aws/aws-lambda-dotnet#454]
When deploying a lambda function for the first time using the following:
```
dotnet lambda deploy-function dev-api-function –-function-role dev-lambda-role --region us-east-1
```
It always prompts with `Select IAM Role that to provide AWS credentials to your code:` even though I have specified the function-role.

As noted in #225 I tried using `--disable-interactive true` but this doesn't work.

Any ideas on how to prevent this prompt? The commands are being ran as part of CI and need to be able to create functions.

Thanks

**Comment by normj** [comment]
I'm not able to replicate this behavior. What version of Amazon.Lambda.Tools are you running? Could you try providing the full role ARN instead of just the name of the role? I'm wondering if there is a permission issue being badly handled when the code attempts to expand the role name to full arn.

The ARN should look like something like "arn:aws:iam::<account-id>:role/dev-lambda-role"

**Comment by KeirSweeney** [comment]
Hi @normj,

I played around using the other `--function-role` alias `-frole` and noticed that was working. Then noticed that when I copied the command from my text editor (which has font ligatures enabled) it stopped working again. After typing the command out in my console, with the `--function-role` flag, it worked. Seems like the issue was with the copy and paste rather than the parameter itself 🤦‍♂️

Thanks for your assistance, sorry for the inconvenience.
